README.md

```diff
@@ -100,3 +100,13 @@ To make your landwalking experience more pleasant, you can make use of a few adm
 * A `/jump` of about 50 will send you soaring
 * [This map](res/dong_number_map.png) (credit to Danny O) is useful for `/warp` coordinates.
 * `/goto` is useful for more precise teleportation (ie. for getting into Infected Zones, etc.).
+
+# Item commands
```

Comment:
I'd use `###` to make these subheadings of `# "Gameplay"`, not full headings. You could also add `### Movement commands` above the preexisting list.